src/Cart/CartProduct.js

```diff
@@ -0,0 +1,237 @@
+/* eslint-disable max-len */
+import React, { useState, useEffect } from 'react';
+import {
+  View, StyleSheet, Image,
+} from 'react-native';
+import {
+  Text, Button, TextInput,
+} from 'react-native-paper';
+import { PropTypes } from 'prop-types';
+import Config from 'react-native-config';
+import Icon from 'react-native-vector-icons/MaterialIcons';
+
+const Airtable = require('airtable');
+
+const image = require('../assets/imgs/square_logo.png');
+
+const airtableConfig = {
+  apiKey: Config.REACT_APP_AIRTABLE_USER_KEY,
+  baseKey: Config.REACT_APP_AIRTABLE_BASE_KEY,
+};
+
+const base = new Airtable({ apiKey: airtableConfig.apiKey })
+  .base(airtableConfig.baseKey);
+
+const styles = StyleSheet.create({
+  container: {
+    justifyContent: 'center',
+    flexDirection: 'row',
+    borderBottomWidth: 1,
+    height: 125,
+    marginTop: 5,
+    marginBottom: 0,
+  },
+  container2: {
+    flex: 1,
+    alignItems: 'center',
+    justifyConent: 'center',
+    flexDirection: 'column',
+    marginLeft: 0,
+    marginTop: 5,
+    marginRight: 5,
+  },
+  container3: {
+    flex: 1,
+    flexDirection: 'column',
+    marginLeft: 0,
+    marginTop: 16,
+  },
+  quantityContainer: {
+    // flex: 1,
+    flexDirection: 'row',
+    height: 65,
+  },
+  itemName: {
+    fontFamily: 'Verdana',
+    fontStyle: 'normal',
+    fontWeight: 'bold',
+    fontSize: 22,
+    marginRight: 10,
+    marginLeft: 8,
+    marginTop: 0,
+    marginBottom: 0,
+  },
+  quantityBox: {
+    width: 50,
+    padding: '2%',
+    margin: '1%',
+    borderRadius: 5,
+    marginTop: 10,
+    marginLeft: 10,
+    marginBottom: 20,
+    justifyContent: 'center',
+    alignContent: 'center',
+    textAlign: 'center',
+  },
+  itemQuantityType: {
+    fontFamily: 'Verdana',
+    fontStyle: 'normal',
+    fontWeight: 'normal',
+    fontSize: 14,
+    marginBottom: 5,
+    marginLeft: 5,
+    marginTop: 25,
+  },
+  itemQuantity: {
+    fontFamily: 'Verdana',
+    fontStyle: 'normal',
+    fontWeight: 'normal',
+    fontSize: 14,
+    alignContent: 'center',
+  },
+  itemTotalPrice: {
+    fontFamily: 'Verdana',
+    fontStyle: 'normal',
+    fontWeight: 'bold',
+    fontSize: 18,
+    marginBottom: '0%',
+    marginLeft: 100,
+    marginTop: 10,
+  },
+  itemPricePer: {
+    fontFamily: 'Verdana',
+    fontStyle: 'normal',
+    fontWeight: 'normal',
+    fontSize: 12,
+    marginBottom: 0,
+    marginLeft: 100,
+  },
+  removeItemButton: {
+    width: '5%',
+    padding: 0,
+    marginTop: 15,
+    marginLeft: 135,
+    marginRight: 10,
+  },
+  image: {
+    width: 80,
+    height: 80,
+    marginLeft: 5,
+    marginTop: 20,
+    marginBottom: 20,
+  },
+});
+
+export default function CartProduct(props) {
+  const {
+    itemID,
+    refresh,
+    setRefresh,
+    name,
+    price,
+    type,
+    initialQuantity,
+  } = props;
+
+  const [item, setItem] = useState([]);
```

Comment:
this seems unnecessary and can be removed, I remember fixing this line w/ Kaylee? item is never used and setItem doesn't need to be used either